Remove pyhdb (#11469)
DataDog/integrations-core: sap_hana/README.md

### Installation

> The SAP HANA check is included in the [Datadog Agent][2] package but to be able to use this integration you must install the [hdbcli][10] library.
> The installation must be performed manually for licensing reasons.

Reviewer: ```suggestion
The SAP HANA check is included in the [Datadog Agent][2] package. To use this integration, you need to manually install the [hdbcli][10] library.

```

> #### Install library
> ##### Unix

Reviewer: ```suggestion
For Unix:
```
